src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -48,6 +48,7 @@
 <sofie-button-group>
   <sofie-button (click)="cancel()" i18n>global.close.button</sofie-button>
   <sofie-spacer></sofie-spacer>
+  <sofie-button *ngIf="!keyboardMapping" (click)="save(true)" [type]="'STANDARD'" [isDisabled]="!actionTriggerForm.valid" [tooltip]="saveButtonErrorsTooltip" i18n>common.save-and-create-new</sofie-button>
```

Comment:
Since we use a static value for `type`, we can remove the dynamic attribute binding so it becomes `type="STANDARD"`.

Reply:
Fix